src/GraphControl/Control/Grapher.h

```diff
@@ -190,6 +190,106 @@ public
 
         event Windows::Foundation::EventHandler<Windows::Foundation::Collections::IMap<Platform::String ^, double> ^> ^ VariablesUpdated;
         void SetVariable(Platform::String ^ variableName, double newValue);
+        Platform::String ^ ConvertToLinear(Platform::String ^ mmlString);
+		
+        // We can't use the EValTrigUnitMode enum directly in as the property type because it comes from another module which doesn't expose
+        // it as a public enum class.  So the compiler doesn't recognize it as a valid type for the ABI boundary.
+        property int TrigUnitMode
+        {
+            void set(int value)
+            {
+                if (value != (int)m_solver->EvalOptions().GetTrigUnitMode())
+                {
+                    m_solver->EvalOptions().SetTrigUnitMode((Graphing::EvalTrigUnitMode)value);
```

Comment:
These APIs are not yet available correct? Could we get those changes in a PR in the impl project so that we can test this change out?